docs/src/user-guide/getting-started.md

```diff
@@ -93,6 +92,57 @@ Your `.eslintrc.{js,yml,json}` configuration file will also include the line:
 
 Because of this line, all of the rules marked "(recommended)" on the [rules page](/docs/rules) will be turned on.  Alternatively, you can use configurations that others have created by searching for "eslint-config" on [npmjs.com](https://www.npmjs.com/search?q=eslint-config).  ESLint will not lint your code unless you extend from a shared configuration or explicitly turn rules on in your configuration.
 
+## Global install
+
+It is also possible to install ESLint globally rather than locally (using `npm install eslint --global`). However, this is not recommended, and any plugins or shareable configs that you use must be installed locally in either case.
+
+## Manual set up
+
+You can also manually set ESLint up your project.
+
+Before you begin, you must have a `package.json` file already. If you don't, make sure to run `npm init` or `yarn init` beforehand.
+
+1. Install the ESLint package in your project:
+
+   ```shell
+   npm install --save-dev eslint
+   ```
+
+1. Add an `.eslintrc` file in one of the [supported configuration file formats](./configuring/configuration-files#configuration-file-formats).
+
+  ```shell
+  # Create JavaScript configuration file
+  touch .eslintrc.js
+  ```
+
+1. Add configuration to the `.eslintrc` file. Refer to the [Configuring ESLint documentation](./configuring/) to learn how to add rules, environments, custom configurations, plugins, and more.
+
+  ```js
+  // .eslintrc.js example
+  module.exports = {
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+  }
+  ```
+
+1. Lint code using the ESLint CLI:
+
+   ```sh
```

Comment:
This is why the build is failing. Prism doesn't support `sh`, it needs to be `shell` or `bash`.

```suggestion
   ```shell
```